src/imageViewer/webpackModules/imageViewer.tsx

```diff
@@ -117,7 +117,7 @@ export default function ImageViewer({
       const newZoom = zoom + (-deltaY / 100) * zoom;
       const newZoomClamped = Math.min(
         20,
-        Math.max(calculatedScale / 10, newZoom)
+        Math.max(0.01, newZoom)
```

Comment:
This seems like a pretty magic number - why'd you change it?

Reply:
ah i see, wasn't sure if there was a good reason for it, I was just upset by zooming being limited to an ugly number since the default zoom was like 33% so the limit would be 3.47% or so, maybe that was just something i was doing